jina/executors/crafters/nlp/split.py

```diff
@@ -31,28 +31,33 @@ def __init__(self,
         self.punct_chars = punct_chars
         if not punct_chars:
             self.punct_chars = ['!', '.', '?', '։', '؟', '۔', '܀', '܁', '܂', '‼', '‽', '⁇', '⁈', '⁉', '⸮', '﹖', '﹗',
-                                '！', '．', '？', '｡', '。']
+                                '！', '．', '？', '｡', '。', '\n']
         if self.min_sent_len < self.max_sent_len:
             self.logger.warning('the min_sent_len (={}) should be smaller or equal to the max_sent_len (={})'.format(
                 self.min_sent_len, self.max_sent_len))
-        self._slit_pat = re.compile('([{0}])+([^{0}])'.format(''.join(self.punct_chars)))
+        self._slit_pat = re.compile('\s*([^{0}]+)(?<!\s)[{0}]*'.format(''.join(self.punct_chars)))
 
-    def craft(self, raw_bytes: bytes, doc_id: int, *args, **kwargs) -> List[Dict]:
+    def craft(self, raw_bytes: bytes, doc_id: int,
```

Comment:
please reformat this line.

Reply:
Done! I have a habit of breaking all lines longer than 80 characters :grimacing: